godotenv_test.go

```diff
@@ -479,6 +479,28 @@ func TestComments(t *testing.T) {
 	loadEnvAndCompareValues(t, Load, envFileName, expectedValues, noopPresets)
 }
 
+func TestIsInt(t *testing.T) {
+	checkAndCompare := func(s string, expected bool) {
+		if isInt(s) != expected {
+			t.Fail()
+		}
+	}
+
+	// invalid values
+	checkAndCompare("", false)
+	checkAndCompare("+123", false)
+	checkAndCompare("+12a3", false)
+	checkAndCompare("12a3", false)
+	checkAndCompare("abc", false)
+	checkAndCompare("12 3", false)
+	checkAndCompare("-", false)
+	checkAndCompare(" ", false)
+
+	// valid values
+	checkAndCompare("-123", true)
+	checkAndCompare("123", true)
```

Comment:
Please add a test about a very long number that would break max int64

It's part of your code change

Reply:
The update is here